### ISSUE TITLE
Space is displayed publicly although it should only be visible to members

### ISSUE THREAD

**Issue by RoundAboutWEB** [humhub/humhub#7083]
The visibility of the Space is set to "Public (registered users only)", but is still displayed under "/spaces/".

| Q                | A
| ---------------- | ---
| HumHub version   | 1.16.0
| PHP version      | 8.2
| Operating system | Linux

**Comment by marc-farre** [comment]
I confirm that the space is visible in the space directory.
However, when clicking on it, the "Login required" page is displayed.

**Comment by RoundAboutWEB** [comment]
It is correct that you can only see the content after logging in, but "Public (registered users only)" means that this space is not displayed in the overview for guests (without login), but is only visible to members.

**Comment by marc-farre** [comment]
@RoundAboutWEB I understood the issue, but I'm not totally sure of which behavior has been decided for HumHub.
@luke- will tell us if it should be fixed or not.

**Comment by dantefromhell** [comment]
I agree with @RoundAboutWEB request that spaces that are marked for "registered users only" raise the expectation to be invisible to guest users.

Configuring a space as visible to guest but default content visibility to private  gives us the possibility to display a spaces existence publicly without making all content publicly visible.

**Comment by luke-** [comment]
Is the Space behaviour currently identical to user profiles? We show the existence, but no further information.

**Comment by RoundAboutWEB** [comment]
According to the title, the Space should not be visible to guests, which would also make sense so that only members (after login) can see the corresponding Space.

If the existence of a Space should be visible to guests, but the content should not, then define the Space as public and the content as "Private". In my opinion, it is therefore a mistake that a Space that is not visible to guests is displayed in the overview.
